src/hull.js

```diff
@@ -172,7 +172,7 @@ function hull(pointset, concavity, format) {
         maxEdgeLen = concavity || 20;
 
     if (pointset.length < 4) {
-        return pointset.slice();
+        return pointset.concat([pointset[0]]).slice();
```

Comment:
concat() already returns a new array so slice() call can be removed from here. Please don't forget to regenerate dist/hull.js after this change.

Reply:
I removed the slice() and also checked the point count after filtering duplicates.